packages/react/src/components/TextField/TextField.types.ts

```diff
@@ -75,6 +75,13 @@ export interface ITextFieldProps extends React.AllHTMLAttributes<HTMLInputElemen
    */
   autoAdjustHeight?: boolean;
 
+  /**
+   * Callback for when text field height is adjusted
+   * This is called for multiline fields for which autoAdjustHeight is true.
+   * @defaultvalue false
```

Comment:
Some minor suggestions for the comment
```suggestion
   * Callback for when text field height is adjusted.
   * This is only called for multiline fields for which `autoAdjustHeight` is true.
```